### ISSUE TITLE
App losing com with softener only when integration is enabled

### ISSUE THREAD

**Issue by dannieboiz** [arturzx/homeassistant-iqua-softener#18]
If I have the integration enabled, the app will no longer see the softener and HA can't pull any data. It takes half a day or a day before it happens. For the first week or so it did it every day. I disabled the integration for the past 3 days and the app is working fine.

**Comment by KajetanKubik** [comment]
> If I have the integration enabled, the app will no longer see the softener and HA can't pull any data. It takes half a day or a day before it happens. For the first week or so it did it every day. I disabled the integration for the past 3 days and the app is working fine.

What do you mean by "app" - mobile application? So either mobile app can work, or HA integration, but not both ? I also have some problems recently, wondering what can be cause.

**Comment by dannieboiz** [comment]
> > If I have the integration enabled, the app will no longer see the softener and HA can't pull any data. It takes half a day or a day before it happens. For the first week or so it did it every day. I disabled the integration for the past 3 days and the app is working fine.
> 
> What do you mean by "app" - mobile application? So either mobile app can work, or HA integration, but not both ? I also have some problems recently, wondering what can be cause.

I don't know if it doesn't work the other way around. I've disabled the HA integration since making this post and  the mobile app has not lost its communication once. I wonder if the iqua doesn't like the constant polling with the integration.

**Comment by rogales** [comment]
Hold on. You mean Iqua APP doesnt see Your softener right?
It's because Your HA refreshes every 5 minutes new informations from Iqua servers which are not used to work  with such viewing frequency. ;)

**Comment by dannieboiz** [comment]
> Hold on. You mean Iqua APP doesnt see Your softener right? It's because Your HA refreshes every 5 minutes new informations from Iqua servers which are not used to work with such viewing frequency. ;)

Correct, and when this happens HA also no longer work until I disable HA. Any work around?

**Comment by rogales** [comment]
You don't need to disable Your HA Softener plugin.
All You have to do is to open and edit refresh frequency in **homeassistant\custom_components\iqua_softener\sensor.py**
row number 41 - UPDATE_INTERVAL = timedelta(minutes=10) (by default it is set to 5 minutes (which is too fast).

**Comment by KajetanKubik** [comment]
> You don't need to disable Your HA Softener plugin. All You have to do is to open and editrefresh frequency in **homeassistant\custom_components\iqua_softener\sensor.py** row number 41 - UPDATE_INTERVAL = timedelta(minutes=10) (by default it is set to 5 minutes (which is too fast).

Sounds good, thanks, will try it.

**Comment by jakubkotlorz** [comment]
Same issue here. I remember also that viessmann guy, installing the device on my site, mentioned that messing up with some "other apps" or "smart home" is causing trouble.
I'll be glad to try your way.

**Comment by dannieboiz** [comment]
Mine was set to 5 sec, changed ít to 5 minutes seems to do the trick

**Comment by petrstehlik** [comment]
Hey everyone, just had a new Aquahome softener installed. Did a few small updates to the integration including prolonging the interval to 10 mins - https://github.com/petrstehlik/homeassistant-iqua-softener
